test/e2e/specs/fakerp/setcontainerimage.go

```diff
@@ -0,0 +1,64 @@
+package fakerp
+
+import (
+	"context"
+	"io/ioutil"
+	"os"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	"github.com/Azure/go-autorest/autorest/to"
+	"github.com/ghodss/yaml"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	pluginapi "github.com/openshift/openshift-azure/pkg/api/plugin"
+	"github.com/openshift/openshift-azure/test/clients/azure"
+	"github.com/openshift/openshift-azure/test/sanity"
+)
+
+var _ = Describe("Change a single image to latest E2E tests [ChangeImage][Fake][LongRunning]", func() {
+	var (
+		azurecli *azure.Client
+		ctx      = context.Background()
+	)
+
+	BeforeEach(func() {
+		var err error
+		azurecli, err = azure.NewClientFromEnvironment(false)
+		Expect(err).NotTo(HaveOccurred())
+	})
+
+	It("should be possible for an SRE to update a single container image", func() {
+		By("Reading the internal config before the image update")
+		before, err := azurecli.OpenShiftManagedClustersAdmin.Get(ctx, os.Getenv("RESOURCEGROUP"), os.Getenv("RESOURCEGROUP"))
+		Expect(err).NotTo(HaveOccurred())
+		Expect(before).NotTo(BeNil())
+
+		By("Executing a cluster update with updated image.")
+
+		data, err := ioutil.ReadFile("../../pluginconfig/pluginconfig-311.yaml")
+		Expect(err).NotTo(HaveOccurred())
+		Expect(data).NotTo(BeNil())
+		var template *pluginapi.Config
+		if err := yaml.Unmarshal(data, &template); err != nil {
+			Expect(err).NotTo(HaveOccurred())
+		}
+
+		before.Config.Images.WebConsole = to.StringPtr(template.Versions[template.PluginVersion].Images.WebConsole)
```

Comment:
here instead of reusing before, construct a brand new admin API object, leave all the pointers at nil except for .Config.Images.WebConsole.

Reply:
ok